### PR TITLE
change login, update readme

### PR DESCRIPTION
**Pull request by StephenCoady**
JIRA 
https://issues.jboss.org/browse/RHMAP-16632

Changes
Update the oc login role to cover all possible cases. Updated the readme to reflect these changes.

### CODE REVIEW

**Comment by StephenCoady** [comment]
@laurafitzgerald review please

File: README.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ Note that `mkdir` and `chmod` commands above should be executed in the Docker-ma
 
 You can find templates for creating PV's backed by glusterFS or NFS in pv-sample-templates/android-sdk-linux and pv-sample-templates/jenkins
 
+### Notes about logging in to the master node
```
Comment (laurafitzgerald) [inline README.md]: could we also add a note about the `skip_tls` option

**Comment by StephenCoady** [comment]
@laurafitzgerald sorry forgot that part, I've edited the readme now.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,22 @@ Note that `mkdir` and `chmod` commands above should be executed in the Docker-ma
...
+2. If you are running the playbook remotely (i.e. while in an ssh session) then master_url does not need to be set as the default address for oc login is https://localhost:8443.
+
+#### Using insecure connections
+
+If you wish to use an insecure connection you can pass the skip_tls environment variable which will bypass the certificate check when logging in to openshift. 
+
+To pass it with the ansible-playbook, it should look like:
+
+```
+-e skip_tls=true
```
Comment (laurafitzgerald) [inline README.md]: * -e "skip_tls=true"

**Comment by laurafitzgerald** [comment]
@StephenCoady small comment but once that's resolved feel free to merge.

**Pull request merged**